### ISSUE TITLE
The problem of remote method invocation

### ISSUE THREAD

**Issue by Jeongwonkyu** [hikalkan/scs#21]
Hi ~
I had some test with your scs about RMI.
The client can request historical data to server then this data size is 13,000 bytes ~ 140,000 bytes almost.
After client request data several times, server memory is increase and die.

Did you have a test about this case ?

**Comment by hikalkan** [comment]
You say there is a memory leak. I used SCS for very long running processes (not restarted for months) but I didn't see such a problem.
This maybe related to garbage collecter which collects object periodically. So, when you call a method it's normal to increase memory and not decrease for a time. Did you tested it for long time to understand if really have a memory leak problem.

**Comment by Jeongwonkyu** [comment]
Thank you ~ I solved this problem.
I found out some problems that related to GC.

And I have one more question ~ it doesn't support the duplicated method name in service ?
I wanna use some override functions

**Comment by hikalkan** [comment]
No, it does not support it (because it does not send argument types to find the override). Please use different naming.
